Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN yarn install --frozen-lockfile --production --ignore-optional && \
     yarn run postinstall
 
 HEALTHCHECK --interval=5m --timeout=3s \
- CMD curl --fail http://localhost:8080 || exit 1
+    CMD curl --fail http://localhost:8080 || exit 1
```

Comment:
could you remove the extra whitespace please?

Reply:
Done